pkg/azure/client.go

```diff
@@ -104,6 +104,9 @@ func NewAzClient(subscriptionID SubscriptionID, resourceGroupName ResourceGroup,
 	if err := az.deploymentsClient.AddToUserAgent(userAgent); err != nil {
 		glog.Error("Error adding User Agent to Deployments client: ", userAgent)
 	}
+	if err := az.roleAssignmentsClient.AddToUserAgent(userAgent); err != nil {
+		glog.Error("Error adding User Agent to Deployments client: ", userAgent)
```

Comment:
Logging: is it supposed to be roleAssignment client?

Reply:
good catch. will correct this.